projects/schematics/src/migrations/5_0/removed-public-api-deprecations/removed-public-api-deprecations.ts

```diff
@@ -1,38 +1,14 @@
 import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
-import {
-  BAD_REQUEST_HANDLER,
-  BAD_VOUCHER_REQUEST_HANDLER,
-  CONFIGURATOR_EVENT_LISTENER,
-  HANDLE_VOUCHER_OPERATION_ERROR,
-} from '../../../shared/constants';
-import {
-  SPARTACUS_CORE,
-  SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
-} from '../../../shared/libs-constants';
 import { DeprecatedNode } from '../../../shared/utils/file-utils';
 import { removedPublicApiDeprecation } from '../../mechanism/removed-public-api-deprecations/removed-public-api-deprecation';
 
-export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
-  // feature-libs/product-configurator/rulebased/core/event/rulebased-configurator-event.listener.ts
-  {
-    node: CONFIGURATOR_EVENT_LISTENER,
-    importPath: SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
-    comment: `'${CONFIGURATOR_EVENT_LISTENER} has been removed and is no longer part of the public API. Please use 'ConfiguratorRouterListener' instead`,
-  },
-  // projects/core/src/global-message/http-interceptors/handlers/bad-request/bad-request.handler.ts
-  {
-    node: BAD_REQUEST_HANDLER,
-    importPath: SPARTACUS_CORE,
-    comment: `'${HANDLE_VOUCHER_OPERATION_ERROR} has been removed and is no longer part of the public API. Please use new methods in ${BAD_VOUCHER_REQUEST_HANDLER}`,
```

Comment:
I think this should have been part of methods and properties instead. 

Should include a migration comment to say to use `method` from cart lib

Reply:
done in https://github.com/SAP/spartacus/pull/15974/commits/c8a9eaced748421f685bffd9801ba06cf627dfbb#diff-b7ec6fc5f3c8128cb7021b4d5c74ab0b9367b60ef72b6a4a3fdfadee563ac207L514

so question now is do we remove this?

If so, I can approve this PR after removing this change.